### PR TITLE
PPPL Department name change from PS&T to Discovery Plasma Science

### PR DESCRIPTION
**Pull request by bess**
Ref #1496 

Screenshot 2023-09-12 at 8 44 20 AM
Screenshot 2023-09-12 at 8 40 43 AM
Screenshot 2023-09-12 at 8 39 20 AM

### CODE REVIEW

File: lib/tasks/metadata.rake [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "byebug"
```
Comment (carolyncole) [inline lib/tasks/metadata.rake:3]: I'm guessing this is not needed
```suggestion
```

**Review by carolyncole — DISMISSED** [review]
Small update if you want to, but looks good to me otherwise...

**Comment by bess** [comment]
@astrochun When we run the rake task (documented here: https://github.com/pulibrary/pdc_describe/blob/70da0f069b120110f3ccfaccf39837c6b538ac11/docs/bulk_metadata_updates.md) that will update all of the records in the system, regardless of whether they are in review or have already been published.

**Comment by astrochun** [comment]
> @astrochun When we run the rake task (documented here: https://github.com/pulibrary/pdc_describe/blob/70da0f069b120110f3ccfaccf39837c6b538ac11/docs/bulk_metadata_updates.md) that will update all of the records in the system, regardless of whether they are in review or have already been published.

Great!

File: lib/tasks/metadata.rake (hunk 1 above)
Comment (bess) [reply lib/tasks/metadata.rake]: 👍🏻 Fixed

**Review by carolyncole — APPROVED** [review]

**Pull request merged**